wurst/__legacy__/lib/ID.jurst

```diff
@@ -882,7 +882,8 @@ library ID requires ObjectIdGenerator, AdvanceIDs
 		public constant integer ITEM_ACID_BOMB               = 'I04K'
 		public constant integer ITEM_ANTI_MAGIC_POTION       = 'I02M'
 		public constant integer ITEM_ANABOLIC_POTION         = 'I02H'
-		public constant integer ITEM_CURE_ALL                = 'I03G'
+		public constant integer ITEM_CURE_ALL                = compiletime(ITEM_ID_GEN.next())
+		// public constant integer ITEM_CURE_ALL                = 'I03G'

```

Comment:
Just delete commented code. You can use the git history if you really need to see the old version.